Examples/Copy-NsxIpset.ps1

```diff
@@ -0,0 +1,57 @@
+function Copy-NsxIpSet {
```

Comment:
Can you add any version or module requirements at the top here. Maybe take a look at the following as an example
https://github.com/vmware/powernsx/blob/master/Examples/EnableFirewallRuleLogging.ps1

Also please add Author details, contact information and script/function/cmdlet versioning